packages/core/src/components/table/table-header-cell/table-header-cell.tsx

```diff
@@ -41,8 +41,8 @@ export class TdsTableHeaderCell {
   /** Enables sorting on that column */
   @Prop() sortable: boolean = false;
 
-  /** Setting for text align, default is left. Other accepted values are "right" or "end". */
-  @Prop({ reflect: true }) textAlign?: string;
+  /** Setting for text align, default is "center". Other accepted values are "left", "start", "right" or "end". */
+  @Prop({ reflect: true }) textAlign: 'left' | 'start' | 'right' | 'end' | 'center' = 'center';
```

Comment:
```suggestion
  @Prop({ reflect: true }) textAlign: 'left' | 'start' | 'right' | 'end' | 'center' = 'left';
```

Use 'left' as default.